MC_data_new.cxx

```diff
@@ -1504,7 +1509,7 @@ void compute_cmos_with_saturation(vector<double>& x_hits_tr,
                     int zz = key % N ;
                     int yy = ((key - zz) / N) % M;
                     int xx = (key - yy * N - zz) / N / M;
-                    hout[xx][yy]+=Nph_saturation(val, optA, optbeta);
+                    hout[xx][yy]+=Nph_saturation(val, optA, optbeta*xy_vox_scale*xy_vox_scale*0.1/zbin_dim);
```

Comment:
Do we want to add a comment like //optbeta is multiplied by factors to normalize on volume chosen ?